cc.arduino.arduinoide2.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>cc.arduino.arduinoide2.desktop</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
this is not the prefered license for the metadata.

Reply:
what is the preferred license? i should be able to change it since i basically wrote it myself.

Reply:
`CC0-1.0`

Also this should be, along the .desktop and proper icons, submitted upstream.
(the lack of icon and .desktop is a definitive bug)

with some luck it can be in the final release.

Reply:
yeah hopefully there is a pr that covers some of this: https://github.com/arduino/arduino-ide/pull/356

Reply:
commented there. But it's missing the appstream file.